### PR TITLE
Make sure to path sequence id in FW rules

### PR DESCRIPTION
**Pull request by annakhm**
Otherwise rule order may be compromised.

Signed-off-by: Anna Khmelnitsky <akhmelnitsky@vmware.com>

### CODE REVIEW

File: nsxt/policy_common.go [hunk 1]
```diff
@@ -437,7 +437,10 @@ func getPolicyRulesFromSchema(d *schema.ResourceData) []model.Rule {
 			Services:             getPathListFromMap(data, "services"),
 			Scope:                getPathListFromMap(data, "scope"),
 			Profiles:             getPathListFromMap(data, "profiles"),
-			SequenceNumber:       &sequenceNumber,
+		}
+
+		if sequenceNumber > 0 {
+			elem.SequenceNumber = &sequenceNumber
```
Comment (ksamoray) [inline nsxt/policy_common.go:443]: So what would be the value of elem.SequenceNumber if sequenceNumber == 0? Wouldn't it be the same?
Should this just protect from negative values in sequenceNumber?

**Review by ksamoray — APPROVED** [review]
LGTM, just a nit/curiosity

File: nsxt/policy_common.go (hunk 1 above)
Comment (annakhm) [reply nsxt/policy_common.go:443]: The value will be nil, which translates to not passing the value in request body. In this case NSX assigns the value, which is what we want.

**Pull request merged**